src/streaming/streamingClient.ts

```diff
@@ -223,7 +223,7 @@ export class StreamingClient {
                 clearInterval(intervalId);
                 subscriptionResolve({
                   runId: this.subscribedTestRunId,
-                  queueItem: result
+                  queueItem: result,
                 });
               }
             }, RetreiveResultsInterval);
```

Comment:
Found this spelling mistake in the constant name.  Rename to `RetrieveResultsInterval`?